.github/actions/install-node-package/action.yml

```diff
@@ -1,8 +1,10 @@
 name: Install dependencies
+description: Install npm dependencies from package.json
 
 inputs:
-  node-version: { required: true }
-  couchdb-version: { required: false, default: '3.1' }
+  node-version:
+    description: Specify the Node.js version to install
+    required: true
```

Comment:
As above - not sure this needed to change.

Reply:
Same Answer as above ;)